scripts/ci_setup.py

```diff
@@ -273,8 +273,8 @@ def __main(args: list) -> int:
 
     # On non-windows platforms, delete dotnet, so that we don't have to deal with chmoding it on the helix machines
     # This is only necessary for netcoreapp3.0 and netcoreapp5.0
-    if sys.platform != 'win32' and remove_dotnet:
-        dotnet.remove_dotnet(architecture)
+    #if sys.platform != 'win32' and remove_dotnet:
+        #dotnet.remove_dotnet(architecture)
```

Comment:
Can we remove this instead of comment it?

Reply:
Yes I submitted another PR #1282  for this issue and will leave it as original for now. Created another issue for now because I'd like to investigate in another seemingly unnecessary download of dotnet cli along with that.